src/input/Input.js

```diff
@@ -969,6 +969,41 @@ Phaser.Input.prototype = {
 
     },
 
+    /**
+    * Releases the active state of the matching Pointer object, passing in the event data.
+    *
+    * @method Phaser.Input#releaseActivePointer
+    * @protected
+    * @param {any} event - The event data from the Touch event.
+    * @return {Phaser.Pointer} The Pointer object that was stopped or null if no Pointer object is available.
+    */
+    releaseActivePointer: function (event)
+    {
+
+        if (this.pointer1.active && this.pointer1.identifier === event.identifier)
+        {
+            return this.pointer1.releaseActive(event);
+        }
+
+        if (this.pointer2.active && this.pointer2.identifier === event.identifier)
+        {
+            return this.pointer2.releaseActive(event);
+        }
+
+        for (var i = 2; i < this.pointers.length; i++)
```

Comment:
Could you document why this starts at 2?

Reply:
The code was copied from an existing Phaser function. This has been refactored as an optional parameter to the existing functions (stop, stopPointer).